custom-sign-in/samples.config.js

```diff
@@ -10,13 +10,15 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { CLIENT_ID, REDIRECT_URI, LOGOUT_REDIRECT_URI, ISSUER } from '@env';
+
 export default {
   oidc: {
-    clientId: '{clientId}',
-    redirectUri: '{customUriScheme}:/loginCallback',
-    endSessionRedirectUri: '{customUriScheme}:/logoutCallback',
-    discoveryUri: 'https://{yourOktaDomain}/oauth2/default',
+    clientId: CLIENT_ID,
```

Comment:
What's the behavior before? Devs need to change the config manually? 
How about also add comments to explain what each config should look like?

Reply:
Added